content/_code-samples/escrow/js/finish-escrow.js

```diff
@@ -0,0 +1,94 @@
+if (typeof module !== "undefined") {
```

Comment:
why don't you just import anyways instead of checking?

Reply:
- https://github.com/XRPLF/xrpl-dev-portal/blob/master/content/_code-samples/issue-a-token/js/issue-a-token.js
- https://github.com/XRPLF/xrpl-dev-portal/blob/master/content/_code-samples/send-xrp/js/send-xrp.js
- https://github.com/XRPLF/xrpl-dev-portal/blob/master/content/_code-samples/freeze/js/set-global-freeze.js
etc...
Initially, I thought I'd simply follow the same approach used on other code samples by @mDuo13 for consistency across the board and one of the requirements listed on the bounty work is `must have feature parity with the existing equivalent code samples in other languages.` (I don't know if it relates)